src/main/java/com/qunite/api/security/JwtAuthorizationFilter.java

```diff
@@ -34,14 +44,20 @@ protected void doFilterInternal(@NonNull HttpServletRequest request,
       filterChain.doFilter(request, response);
       return;
     }
-    jwtService.verifyAccessToken(authHeader.substring(7))
-        .ifPresent(decodedJWT -> {
-          var username = decodedJWT.getSubject();
-          var authToken = new UsernamePasswordAuthenticationToken(
-              username, null, Collections.emptyList()
-          );
-          SecurityContextHolder.getContext().setAuthentication(authToken);
-        });
+    try {
+      jwtService.verifyJwt(authHeader.substring(7), TokenType.ACCESS)
+          .ifPresent(decodedJWT -> {
+            var username = decodedJWT.getClaim("username").asString();
+            var authToken = new UsernamePasswordAuthenticationToken(
+                username, null, Collections.emptyList()
+            );
+            SecurityContextHolder.getContext().setAuthentication(authToken);
+          });
+    } catch (RuntimeException exception) {
```

Comment:
Mb specify more clear and less abstract JWTVerificationException instead of RuntimeException?